### ISSUE TITLE
Gymnasium v1.0.0 alphas

### ISSUE THREAD

**Issue by pseudo-rnd-thoughts** [conda-forge/gymnasium-feedstock#33]
### Solution to issue cannot be found in the documentation.

- [X] I checked the documentation.

### Issue

Conda-forge doesn't seem to have created an auto PR for the 1.0.0 alpha 1

### Installed packages

```shell
.
```

### Environment info

```shell
.
```

**Comment by traversaro** [comment]
Indeed probably this is a bug in the version autotick bot, or somehow it detects it is an alpha version.
 Anyhow, note that creating pre-release on conda-forge requires a bit more modifications beside bumping the version in the recipe: see https://conda-forge.org/docs/maintainer/knowledge_base/#pre-release-builds for more details.

**Comment by h-vetinari** [comment]
The bot does not open PRs for pre-release versions, you'll need to open a PR manually. If you want to also release those builds (not just run the CI through its paces), the linked docs are the right place, but it's not very complicated. We'd just need to add:
```
channel_targets:
  - conda-forge gymnasium_dev  # or gymnasium_alpha or ...
```
to `recipe/conda_build_config.yaml` and then rerender. I can create a branch for this, so we don't do have to do these things on `main`, which is supposed to have the latest GA version.

**Comment by pseudo-rnd-thoughts** [comment]
@h-vetinari With v0.29.1 merged, could we make a PR for v1.0.0a1. 

I was intending this as a testing ground for alpha 1 as conda tests on windows (and mac) which we don't have on the main repo so I don't care if we don't end up merging until v1.0.0 is fully done

**Comment by traversaro** [comment]
> @h-vetinari With v0.29.1 merged, could we make a PR for v1.0.0a1.
> 
> I was intending this as a testing ground for alpha 1 as conda tests on windows (and mac) which we don't have on the main repo so I don't care if we don't end up merging until v1.0.0 is fully done

https://github.com/conda-forge/gymnasium-feedstock/pull/36
